db/schema.rb

```diff
@@ -18,21 +17,20 @@
 
   create_table "admins", force: :cascade do |t|
     t.string "css_id", null: false
+    t.index ["css_id"], name: "index_admins_on_css_id", unique: true
```

Comment:
We omitted B-tree, since it's a default index type for PostgreSQL. (ref. https://www.postgresql.org/docs/9.5/static/indexes-types.html)